maven-enforcer-extension/pom.xml

```diff
@@ -71,6 +71,7 @@
             <artifactId>maven-invoker-plugin</artifactId>
             <configuration>
               <showErrors>true</showErrors>
+              <streamLogs>true</streamLogs>
```

Comment:
please don't use `streamLogs` - `streamLogsOnFailures` is configured in parent and should be enough in error case

Reply:
Not using streamLogs - as requested.